python3Packages.polars: 1.21.0 -> 1.26.0

tanya1866 opened the pull request:
Updated python3Packages.polars: 1.21.0 -> 1.26.0

Changelog: https://github.com/pola-rs/polars/releases/tag/py-1.26.0
Code diff between versions: https://github.com/pola-rs/polars/compare/py-1.21.0...py-1.26.0

Closes #395067
Maintainers: @happysalada @SomeoneSerge @GaetanLepage

happysalada commented:
Im om mobile, so i can only get the latest changelog

https://github.com/pola-rs/polars/releases/tag/py-1.26.0

The only problem i remember with this was with connectorx, but we dont have it packaged in nixpkgs, so should be good to go.
I dont know why the review failed on all paclages, that seems odd.

tanctl committed:
python3Packages.polars: 1.21.0 → 1.26.0

Changelog: https://github.com/pola-rs/polars/releases/tag/py-1.26.0
Diff: https://github.com/pola-rs/polars/compare/py-1.21.0...py-1.26.0

tanya1866 commented:
wait it was already there 
why else is it failing to build? :(

SomeoneSerge commented:
Latest job for `rustc` seems to build `1.85`, the experimental feature seems to have been added in `1.86`: https://github.com/rust-lang/rust/pull/133964#event-15815438812

happysalada commented:
1.86 isnt even released yet, before it makes it to master , its likely another month at least. Lets try again then.

tanya1866 commented:
> 1.86 isnt even released yet, before it makes it to master , its likely another month at least. Lets try again then.

i think i saw a pr today updating rustc to 1.86

dotlambda commented:
already on master
